setup.py

```diff
@@ -28,6 +28,7 @@ def package_files(directory):
     "torch>=1.9",
     "argparse>=1.4.0",
     "yahp>=0.0.14",
+    "custom_inherit==2.3.2",
```

Comment:
this is only used in one place.. can we remove from here to reduce bloat?

Reply:
Yes, will do.